tests/add-error-specs.ts

```diff
@@ -0,0 +1,43 @@
+import { expect } from "chai";
+import { mount, ReactWrapper } from "enzyme";
+
+import { addError, Form } from "../src";
+import { ExampleModel } from "./helpers/ExampleModel";
+
+describe("addError()", () => {
+
+    const context = new Form({
+        instantiate: () => new ExampleModel()
+    }).getChildContext();
+
+    it("Should add error if status equals 400", () => {
+        const error = {
+            response: {
+                data: {
+                    errors: [
+                        {
+                            attribute: "email",
+                            details: "exist"
+                        },
+                        {
+                            attribute: "password",
+                            details: "exist"
+                        }
+                    ]
+                },
+                status: 400
+            }
+        };
+
+        addError(context, error);
+        expect(context.getError("email").details).to.equal("exist");
+        expect(context.getError("password").details).to.equal("exist");
+    });
+
+    it("Should throw error if status not equals 400", () => {
+        const error = "error))"
+
+        expect(() => addError(context, error)).to.throw(error);
+
+    });
+});
```

Comment:
Tests for invalid error (second argument), should be written.
Test cases:
- error.response.data is not object or have no errors property
- error.response.data.errors is not array
- error.response.data.errors[N] is not valid error